### ISSUE TITLE
[HowtoTest] Browser Zoom, Text Resize, Magnification section

### ISSUE THREAD

**Issue by rnguyen253** [tmobile/a11yEngineer#402]
Task: Create a How To page covering the topic of Browser Zoom, Text Resize, Magnification

Topics to include:
Importance of these features
Browser zoom instructions from WCAG test steps https://www.w3.org/WAI/WCAG22/Techniques/failures/F102
Text resize test steps https://www.w3.org/WAI/WCAG22/Techniques/failures/F69
Briefly cover the general idea and use of OS magnification on Windows and Mac.
